Apps/HNClient/src/appsettings.json

```diff
@@ -5,6 +5,7 @@
         }
     },
     "OpenIdConnect": {
+        "RequireHttpsMetadata": "false",
```

Comment:
same as above - should be true or removed.

Reply:
+1. Vote on being removed